README.md

```diff
@@ -166,6 +166,12 @@ aws-adfs integrates with:
     password = your_password
     ```
 
+* .aws/credentials profile for automatically refreshing credentials
+    ```
+    [example-role-ue1]
```

Comment:
```suggestion
* .aws/config profile for automatically refreshing credentials
    ```
    [profile example-role-ue1]
```

Reply:
@pdecat - I must have missed this in the documentation, but funny that it works in the credentials file as originally written! Agreed though, it should match the AWS documentation. Adjusting